docs/en/aws-elastic-serverless-forwarder-configuration.asciidoc

```diff
@@ -288,6 +290,13 @@ inputs:
 
 You can define `inputs.[].expand_event_list_from_field` as a string with the value of a key in the JSON that contains a list of elements that must be sent as events instead of the encompassing JSON.
 
+If you want to inject in the expanded events, fields from the root of the JSON object that you are expanding from, you can define `inputs.[].root_fields_to_add_to_expanded_event`. The config takes two different type of value:
```

Comment:
This description is hard to follow. Would you mind adjusting the example in this topic to include the `root_fields_to_add_to_expanded_event` setting so that I (and users) can better understand what you are trying to say here? Then I can offer edits.

I think the big question I need answered is: What do we mean by root fields? I'm assuming we mean the fields under `Records`, but seeing a realistic example would help me understand. Thanks!

Reply:
@dedemorton the example is present: https://github.com/elastic/elastic-serverless-forwarder/pull/642/files/5d1d09b9a65bafee56ce49a4dccd434e9ed96c98#diff-63801152050878371d35100604aa7d8d328f9d97bdecba179bb617554bbae968R280-R281 :)


The root fields are the fields at the root of the JSON object that's expanded, so at the same level for `Records`, for example:
```
{ "Records": [...], "owner": 1234567890, "logGroup": "a log group", "logStream": "a log stream" }
```

Reply:
@aspacca I didn't mean the configuration example. I meant it would help to see how the `root_fields_to_add_to_expanded_event` setting affects the data that is shown under the [Example section](https://www.elastic.co/guide/en/esf/current/aws-elastic-serverless-forwarder-configuration.html#_example) in the docs (where it shows how the setting affects the events that are produced).